datman/config.py

```diff
@@ -515,14 +518,14 @@ def series_map(self):
                 pattern = self.get(tag, 'Pattern')
             except KeyError:
                 raise KeyError("Cant retrieve 'Pattern' from config. Did you "
-                        "specify a site?")
+                               "specify a site?")
             if type(pattern) is list:
                 pattern = "|".join(pattern)
             series_map[tag] = pattern
         return series_map
 
     def keys(self):
-        return self.tags.keys()
+        return list(self.tags.keys())
```

Comment:
Same as other comments regarding letting python-ducktyping do it's thing

Reply:
Same as above. At least some of the calls to this function need a list and not a generator.

Reply:
```suggestion
        return list(self.tags)
```
As mentioned in similar comment above. Feel free to disagree haha